config.py

```diff
@@ -0,0 +1,32 @@
+import logging
+import os
+
+
+class Config(object):
+    ENV_TYPE = os.getenv("ENV_TYPE")
+    JSON_LOGGING = (os.getenv("JSON_LOGGING", 'false').lower() == 'true')
+    LOGGING_LEVEL = logging.INFO
+    LANGUAGES = os.getenv('APE_LANGUAGES', 'en,fr,pt_PT').split(',')
+    DEFAULT_LANGUAGE = os.getenv('APE_DEFAULT_LANGUAGE', 'en')
+    AUTH0_DOMAIN = os.getenv('AUTH0_DOMAIN', 'fake-auth0-domain')
+    AUTH0_CLIENT_ID = os.getenv('AUTH0_CLIENT_ID', 'client_id')
+    AUTH0_CLIENT_SECRET = os.getenv('AUTH0_CLIENT_SECRET', 'client_secret')
+    APP_SECRET_KEY = os.getenv('APP_SECRET_KEY', '1234567890abcdef')
```

Comment:
Fake values here is not a good idea. The app will fail later down the lane in local dev and more importantly in production.
I suggest using standard None values here, e.g. `AUTH0_DOMAIN = os.getenv('AUTH0_DOMAIN')` and catching it later with an exception in app initialization.
```
if not app.config.AUTH0_COMAIN:
  raise ApeConfigError("Missing Auth0 configuration")
```